### PR TITLE
OSDOCS-4057: OCP on ARM release notes for 4.12

### PR DESCRIPTION
**Pull request by kelbrown20**
For Version 4.12
[OSDOCS-4057](https://issues.redhat.com//browse/OSDOCS-4057)

Preview: [Release notes -> Installation and upgrade ](https://52500--docspreview.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-12-release-notes.html#ocp-4-12-installation-and-upgrade)

### CODE REVIEW

**Comment by kelbrown20** [comment]
@jeffdyoung Hey Jeff! Can you PTAL at this PR when you get the chance?

File: release_notes/ocp-4-12-release-notes.adoc [hunk 1]
```diff
@@ -88,6 +88,14 @@ Beginning with {product-title} {product-version}, you can enable a feature set a
 
 For more information about enabling a feature set during installation, see xref:../nodes/clusters/nodes-cluster-enabling-features.adoc#nodes-cluster-enabling[Enabling {product-title} features using feature gates].
 
+[id="ocp-4-12-arm-on-azure"]
+==== {product-title} on ARM
+{product-title} {product-version} is now supported on ARM architecture based Azure installer-provisioned infrastructure. AWS Graviton 3 processors are now available for cluster deployments and is also supported on {product-title} 4.11. For more information about instance availability and installation documentation see, see xref:../installing/installing-preparing.html#supported-installation-methods-for-different-platforms[Supported installation methods for different platforms]
```
Comment (tmalove) [inline release_notes/ocp-4-12-release-notes.adoc:93]:
```suggestion
{product-title} {product-version} is now supported on ARM architecture-based Azure installer-provisioned infrastructure. AWS Graviton 3 processors are now available for cluster deployments and are also supported on {product-title} 4.11. For more information about instance availability and installation documentation, see xref:../installing/installing-preparing.html#supported-installation-methods-for-different-platforms[Supported installation methods for different platforms]
```

**Comment by aleskandro** [comment]
lgtm

**Pull request merged**